develop-docs/README.md

```diff
@@ -128,3 +128,14 @@ iOS 12 opens a risk of introducing bugs, which has already happened in the past,
 the iOS 12 simulator a try.
 
 Related to [GH-2218](https://github.com/getsentry/sentry-cocoa/issues/2218)
+
+### Adding Swift code in the project
```

Comment:
Thank you, @brustolin, for adding an entry here, so we know in the future why we decided that way.